packages/client/components/PokerCardDeck.tsx

```diff
@@ -185,6 +191,35 @@ const PokerCardDeck = (props: Props) => {
     }
   }, [maxSlide, isCollapsed])
   // const transform = maxSlide > 0 && !isCollapsed ? `translateX(${swipe.translateX}px)` : undefined
+
+  const onKeyDown = (event: Event | KeyboardEvent) => {
+    // Ignore action if the card list is empty
+    if (cards.length <= 0) return
+    // When the up key is pressed
+    if ((event as KeyboardEvent).key === 'ArrowUp') {
+      if (typeof selectedIdx === 'undefined' || selectedIdx === cards.length - 1) {
+        // Select the left-most card if no card is selected or if the right-most card was previously selected
+        vote((cards[0] as Card).label)
+      } else {
+        // Otherwise select the card to the right
+        vote((cards[selectedIdx + 1] as Card).label)
+      }
+    }
+
+    // When the down key is pressed
+    if ((event as KeyboardEvent).key === 'ArrowDown') {
+      if (typeof selectedIdx === 'undefined' || selectedIdx === 0) {
+        // Select the right-most card if no card is selected or if the left-most card was previously selected
+        vote((cards[cards.length - 1] as Card).label)
+      } else {
+        // Otherwise select the card to the left
+        vote((cards[selectedIdx - 1] as Card).label)
+      }
+    }
+  }
+
+  window.onkeydown = onKeyDown
```

Comment:
-1 we'll want to clean up the function on unmount. Consider using https://github.com/ParabolInc/parabol/blob/f3361ab138d76ecbd2fa233e518790226414709e/packages/client/hooks/useHotkey.ts#L7

Reply:
Done @mattkrick